### ISSUE TITLE
Edit labels of shortcuts and apps

### ISSUE THREAD

**Issue by philwall2** [BranchMetrics/sesame_issues#14]
Users want to be able to rename shortcuts so they can search for them how they like.

NTH: be able to add multiple labels. But why not just make the main label say everything you want?

NTH: edit icons

NTH: create tags you can assign to shortcuts

**Comment by resolritter** [comment]
I know this is already being worked on or at least being considered, but let me share another case where this is needed.

I have both Firefox and Firefox Nightly installed. Sometimes I want to reach for the first, but Nightly shadows it and makes it impossible to do it through the keyboard alone.

 ![pic](https://i.imgur.com/qZZcwpZ.jpg)

If I could rename them to FF and FN, I'd not only save some keystrokes, but also be able to reliably choose one or the other.

**Comment by philwall2** [comment]
Makes sense. I'll move this up our internal roadmap.

**Comment by real-mizuya** [comment]
It's possible in Nova, it would be nice to have it in Sesame too ❤️

**Comment by resolritter** [comment]
Still having the issue I described before, this time between "FX" and "FX Text Edit". I want to be able to give specific names to a couple of apps to make them easier to remember. @philwall2 Is it coming in a near release?

**Comment by philwall2** [comment]
@reaysawa not really. We spent a ton of time building Sesame into a library for launchers integrations. Now we're working on a new data feature for more contact actions. It will be a few more months before we get back to UI features.

**Comment by Francesco270** [comment]
This would be very useful!

**Comment by Heronheart** [comment]
I would like to bump this issue.  Many apps have very unintuitive names.  For instance I would like to be able to search for "reddit" to bring up "Sync Pro".

**Comment by HEPBO3AH** [comment]
I'd love to see this as well!

**Comment by HunterArmstrong** [comment]
YESS!!!

**Comment by HEPBO3AH** [comment]
Did it fall on any timeline by now?

**Comment by crackers8199** [comment]
add me to the list...for example, Facebook messenger lite is just named "lite," so I can't find it if I search for messenger. I have it renamed in Nova, but obviously can't yet do that in sesame.

**Comment by Progaros** [comment]
Same problem here with some apps. 
One example would be "Amazon" and "Amazon Visa".
If I could rename them to "Amazon" and "Visa" it would be a lot easier to open the right one every time.

**Comment by resolritter** [comment]
I like Sesame, but the fact that this feature has been forsaken after all time is a huge let down. The main point of Sesame for me is to quickly open the application I want through text filtering, without having to stop typing in order to tap anything - so much that even if it didn't have anything else, I'd still use it for this reason alone.

I saw that there were some announcements for integrations, voice search or whatnot... While I don't use those other things (and don't plan to do it in the future), I understand that they might be useful for some. Still, it's weird to me that core UX improvements are not given the main priority. i.e. I think it's unreasonable that development time has gone somewhere else and this core improvement has been forsaken for so long.

I voiced my complaints back in 2019. I've been using Sesame since then and this pain point often pops up on a daily basis, so much that I also remember this issue from time to time. I think you owe us a commitment and date for this @philwall2 @stevehb , or at least say that you are not going to implement it if that's the case.
